extension/js/common/composer.ts

```diff
@@ -1620,6 +1621,24 @@ export class Composer {
       }
       this.debug(`input_to.blur -> parseRenderRecipients done`);
     }));
+    this.S.cached('input_to').bind('paste', Ui.event.handle(async (elem, event) => {
+      if (event.originalEvent instanceof ClipboardEvent && event.originalEvent.clipboardData) {
+        const textData = event.originalEvent.clipboardData.getData('text/plain');
+        const keyImportUi = new KeyImportUi({ checkEncryption: true });
+        let normalizedPub: string;
+        try {
+          normalizedPub = await keyImportUi.checkPub(textData);
+        } catch (e) {
+          return; // key is invalid
+        }
+        const { keys: [key] } = await Pgp.key.parse(normalizedPub);
+        const keyUser = Str.parseEmail(key.users[0]);
```

Comment:
Everything looks done very well. The only issue is that some keys may be lacking users in it. We don't support such keys. But in this particular case, you would probably get an exception because `key.users[0]` may be undefined. Therefore I would add one more check and return early if you find `!key.users.length`.

Reply:
Okay, Got it, thanks.

I thought there is always should be a user.